docs/_partner_api/authentication.md

```diff
@@ -0,0 +1,96 @@
+|    **client_id**    |  Required   | Your application ID.                                                                                                                                                                                                |
+|  **redirect_uri**   |  Required   | The URI you provided in the sign-up form. If you need to modify this, you'll need to [send us a request](mailto:partners@zapier.com).                                                                               |
+|  **response_type**  |  Required   | Use `token`.                                                                                                                                                                                                        |
+|      **scope**      |  Optional   | Space (`%20`) separated variable. See each resource for their required scope, if any.                                                                                                                               |
+| **approval_prompt** |  Optional   | One of `auto` or `force`. Use `auto` if the second authorization (before expiration of previous token) should not prompt the user to re-authorize. Use `force` if the user should authorize your application again. |
+|      **state**      |  Optional   | A unique string to help your application guard against XSRF.                                                                                                                                                        |
+
+#### Example Prompt
+
+![Example OAuth2 Authorization Prompt](https://cdn.zapier.com/storage/photos/d926ea5ba6ca80c184a45cf3f5e420fb.png)
+
+### Receiving the Token, or Error
+
+If the user cancels, or approves the authorization the user will be redirected to your `redirect_uri` with the following example urls:
+
+**Approved**
+```
+http://your.redirect.url/#access_token=iuqhw8egojqenduvybtoken_type=Bearer&expires_in=36000&scope=zap
+```
+
+**Cancelled**
+```
+http://your.redirect.url/?error=access_denied
+```
+
+Your application should use JavaScript to parse the hash parameter and use the token as needed. The **access token will not expire**. If ever invalid, however, provide the user with the authorize flow once more. In the `implicit` grant type, there are no refresh tokens. You can use a hidden iframe with `approval_prompt=auto`, or ask the user to authorize once more, to receive new tokens.
+
+
+### Using the token:
+Preferred use of the tokens is via an HTTP Authorization Header.
+
+```bash
+curl -H "Authorization: Bearer {token}" "https://api.zapier.com/v1/zaps"
+```
+
+
+### Workaround for Implicit Only
+
+While we consider and implement other OAuth flows. The following is a suggested workaround for working with the implicit OAuth flow. The following is a sequence diagram that we'll use to explain the workaround:
```

Comment:
> While we consider and implement other OAuth flows

sounds like it's missing the second half of the sentence. Or, the `.` should be a `,`?